### ISSUE TITLE
Test: new release and automatic package deployment

### ISSUE THREAD

**Issue by pierrepo** [patrickfuchs/buildH#126]
@patrickfuchs we would like to test the automatic deployment of PyPI and conda packages upon new release.

For this, could you please:

- [x] Create a token in PyPI (see [doc](https://github.com/patrickfuchs/buildH/blob/master/devtools/RELEASE.md#setup))
- [x] Store this token in the GitHub secrets for the repo (see [doc](https://github.com/patrickfuchs/buildH/blob/master/devtools/RELEASE.md#setup))
- [x] Update version number (probably minor)
- [x] Create a new release on GitHub

Then verify there are:
- [x] a new version in [Zenodo](https://doi.org/10.5281/zenodo.4676217)
- [x] a new version in [PyPI](https://pypi.org/project/buildh/)
- [x] a new version in [conda](https://anaconda.org/bioconda/buildh)

**Comment by patrickfuchs** [comment]
It worked for Zenodo and Pypi, but not for conda... I waited ~10' after the release and still not there, so I think the automatic deployement didn't work for conda.

**Comment by pierrepo** [comment]
Thanks for testing @patrickfuchs 
Good news for Zenodo and PyPI :tada: 
We should probably wait a couple of hours for conda since it's Bioconda's bot that build the package for us.

**Comment by patrickfuchs** [comment]
Indeed, something happened, I received a notification: https://github.com/bioconda/bioconda-recipes/pull/29200
So I think it worked 👍 
Do you know what we have to do now?

**Comment by pierrepo** [comment]
> Do you know what we have to do now?

I guess nothing. Bioconda maintainers might have to approve this new version.

**Comment by HubLot** [comment]
Yes, someone need to approve the PR so it can be merge. 
Anyway, our automated tools work \o/

**Comment by pierrepo** [comment]
Yep. This is awesome!

**Comment by patrickfuchs** [comment]
Great 😄 So I guess we can close this issue?

**Comment by pierrepo** [comment]
Could we wait some time the conda package is cooked?

**Comment by pierrepo** [comment]
This is so cool!

![image](https://user-images.githubusercontent.com/733861/123334671-b9b36900-d543-11eb-9134-8ea00ab5d3bf.png)

**Comment by patrickfuchs** [comment]
Yes it worked 🎉 !!! Someone from bioconda had to approve before it was merged, that's why it takes more time than Pypi.
Quick question, in the main README and in the section installation, I changed the link to Bioconda so that they point to the buildH page within Bioconda. But I realized there are 2 possible pages :

- main page: https://anaconda.org/bioconda/buildh
- a recipe page: http://bioconda.github.io/recipes/buildh/README.html

In your opinion, which one is more appropriate?
